src/pvrclient-nextpvr.cpp

```diff
@@ -836,11 +894,11 @@ PVR_ERROR cPVRClientNextPVR::GetStreamReadChunkSize(int& chunksize)
 
 bool cPVRClientNextPVR::IsServerStreaming()
 {
-  if (IsServerStreamingLive(false) || IsServerStreamingRecording(false))
+  if (IsServerStreamingLive(false) || m_multistreamRecording.size() > 0)
   {
     return true;
   }
-  kodi::Log(ADDON_LOG_ERROR, "Unknown streaming state %d %d %d", m_nowPlaying, m_recordingBuffer->GetDuration(), !m_livePlayer);
+  kodi::Log(ADDON_LOG_ERROR, "Unknown streaming state %d %d %d", m_nowPlaying, m_multistreamRecording.size(), !m_livePlayer);
```

Comment:
`std::lock_guard<std::recursive_mutex> lock(m_multistreamRecording);` prior to accessing map.

No need to stop the lock, happens automatically at the end block